std/typecons.d

```diff
@@ -799,35 +799,74 @@ if (distinctFieldNames!(Specs))
          *           for comparison between `Tuple`s.
          *
          * Returns:
-         * For any values `v1` on the right-hand side and `v2` on the
-         * left-hand side:
+         * For any values v1 contained by the left-hand side tuple and any
```

Comment:
Sorry for nit picking, but can you use use quotes (`) for expressions: v1, v2, v1 == v2  etc. ?

Reply:
Oh, no nit picking, that's fine. I need to find my way into the D conventions again... Will fix this soon.